packages/app/src/background/services/bandada/BandadaService.ts

```diff
@@ -47,8 +54,17 @@ export class BandadaService {
     throw new Error(result.message.toString());
   }
 
-  async generateMerkleProof({ groupId, commitment }: IGenerateBandadaMerkleProofArgs): Promise<IMerkleProof> {
-    const response = await fetch(`${API_URL}/groups/${groupId}/members/${commitment}/proof`, {
+  async checkGroupMembership({ identity, groupId }: ICheckBandadaGroupMembershipArgs): Promise<boolean> {
+    const response = await fetch(`${API_URL}/groups/${groupId}/members/${hexToBigint(identity.commitment)}`, {
```

Comment:
Why do we need this conversion `hexToBigint`?

Reply:
Bandada needs bigint format. Hex values are not supported.